scripts/MetadataMd.ts

```diff
@@ -303,14 +302,14 @@ function writeFile (name: string, ...chunks: any[]): void {
6e74496414746f706963404174746573746174696f6e546f7069631476616c7565404174746573746174696f6e56616c75650c8c20437265617465206f722075706461746520616e206578697374696e6720636c61696df020546865206069737375657260206f662074686520636c61696d20636f6d65732066726f6d207468652065787472696e73696320606f726967696e603d01205468652060746f7069636020616e64206076616c7565602061726520626f746820553235362077686963682063616e20686f6c6420616e792033322d6279746520656e636f64656420646174612e3072656d6f76655f636c61696d0818686f6c64657230543a3a4163636f756e74496414746f706963404174746573746174696f6e546f70696308f02052656d6f7665206120636c61696d2c206f6e6c7920746865206f726967696e616c206973737565722063616e2072656d6f7665206120636c61696d61012049662074686520606973737565726020686173206e6f742079657420697373756564206120636c61696d206f662060746f706963602c20746869732066756e6374696f6e2077696c6c2072657475726e206572726f722e010c30436c61696d4372656174656410244163636f756e744964244163636f756e744964404174746573746174696f6e546f706963404174746573746174696f6e56616c75650030436c61696d52656d6f7665640c244163636f756e744964244163636f756e744964404174746573746174696f6e546f7069630030436c61696d5570646174656410244163636f756e744964244163636f756e744964404174746573746174696f6e546f706963404174746573746174696f6e56616c75650000001d1c52657761726473011c526577617264731034496e666c6174696f6e5261746501002446697865644931323840000064a7b3b6e00d000000000000000004010120496e666c6174696f6e2072617465202520746f206170706c79206f6e20726577617264207061796f7574732c206974206d6179206265206e656761746976654c446576656c6f706d656e7446756e6454616b6501001c50657262696c6c100000000004f820446576656c6f706d656e742066756e6420252074616b6520666f7220726577617264207061796f7574732c2070617274732d7065722d62696c6c696f6e445472616e73616374696f6e466565506f7401003042616c616e63654f663c543e400000000000000000000000000000000004bc20416363756d756c61746564207472616e73616374696f6e206665657320666f7220726577617264207061796f7574605472616e73616374696f6e466565506f74486973746f727901005856656344657175653c42616c616e63654f663c543e3e040004dc20486973746f72696320616363756d756c61746564207472616e73616374696f6e2066656573206f6e20726577617264207061796f75740108487365745f696e666c6174696f6e5f7261746508246e756d657261746f720c6936342c64656e6f6d696e61746f720c693634045501205365742074686520706572207061796f757420696e666c6174696f6e20726174652028606e756d657261746f7260202f206064656e6f6d696e61746f72602920286974206d6179206265206e6567617469766529647365745f646576656c6f706d656e745f66756e645f74616b6504406e65775f74616b655f70657263656e740c75333204c4205365742074686520646576656c6f706d656e742066756e642074616b6520252c2063617070656420617420313030252e0104305265776172645061796f7574081c42616c616e63651c42616c616e636504b4204120726577617264207061796f75742068617070656e656420287061796f75742c2072656d61696e6465722900001e041c40436865636b5370656356657273696f6e38436865636b547856657273696f6e30436865636b47656e6573697338436865636b4d6f7274616c69747928436865636b4e6f6e63652c436865636b576569676874604368617267655472616e73616374696f6e5061796d656e74';
```

Comment:
Why meta has copied here directly?

Reply:
I had in mind to use it from static meta data file.. have used it here..